src/System.Management.Automation/engine/remoting/common/PSETWTracer.cs

```diff
@@ -198,8 +198,16 @@ internal enum PSEventId : int
     [Flags]
```

Comment:
The enum is actually not a flags enum on Windows ...

Reply:
Fixed